### ISSUE TITLE
Passing a buff to `BraceLabel` breaks in v0.8

### ISSUE THREAD

**Issue by sudgy** [ManimCommunity/manim#1769]
After updating to v0.8, one of my scenes broke.  The following code illustrates the problem:

```python
class Test(Scene):
    def construct(self):
        m = Square()
        bl = BraceLabel(m, "Test", buff = 0.5)
```

This code used to work in v0.7 (and it did what I expected, having a larger distance between the square and the brace), but now in v0.8 it fails with this error:

```
Traceback (most recent call last):
╭──────────────────────────────────────────────────────────────────────────────────────╮
│ File "/home/sudgy/.local/lib/python3.9/site-packages/manim/cli/render/commands.py",  │
│line 135, in render                                                                   │
│    132         for SceneClass in scene_classes_from_file(file):                      │
│    133             try:                                                              │
│    134                 scene = SceneClass()                                          │
│  ❱ 135                 scene.render()                                                │
│    136             except Exception:                                                 │
│    137                 error_console.print_exception()                               │
│    138                 sys.exit(1)                                                   │
│ File "/home/sudgy/.local/lib/python3.9/site-packages/manim/scene/scene.py", line 210,│
│in render                                                                             │
│     207         """                                                                  │
│     208         self.setup()                                                         │
│     209         try:                                                                 │
│  ❱  210             self.construct()                                                 │
│     211         except EndSceneEarlyException:                                       │
│     212             pass                                                             │
│     213         except RerunSceneException as e:                                     │
│ File "/home/sudgy/programs/python/manim/test/test.py", line 6, in construct          │
│    3 class Test(Scene):                                                              │
│    4     def construct(self):                                                        │
│    5         m = Square()                                                            │
│  ❱ 6         bl = BraceLabel(m, "Test", buff = 0.5)                                  │
│ File "/home/sudgy/.local/lib/python3.9/site-packages/manim/mobject/svg/brace.py",    │
│line 148, in __init__                                                                 │
│    145     ):                                                                        │
│    146         self.label_constructor = label_constructor                            │
│    147         self.label_scale = label_scale                                        │
│  ❱ 148         super().__init__(**kwargs)                                            │
│    149                                                                               │
│    150         self.brace_direction = brace_direction                                │
│    151         if isinstance(obj, list):                                             │
│ File "/home/sudgy/.local/lib/python3.9/site-packages/manim/mobject/types/vectorized_m│
│object.py", line 104, in __init__                                                     │
│     101         self.shade_in_3d = shade_in_3d                                       │
│     102         self.tolerance_for_point_equality = tolerance_for_point_equality     │
│     103         self.n_points_per_cubic_curve = n_points_per_cubic_curve             │
│  ❱  104         Mobject.__init__(self, **kwargs)                                     │
│     105                                                                              │
│     106     def get_group_class(self):                                               │
│     107         return VGroup                                                        │
╰──────────────────────────────────────────────────────────────────────────────────────╯
TypeError: __init__() got an unexpected keyword argument 'buff'
```

I'm not exactly sure what changed, but it's pretty obvious what it is that's making it crash.

**Comment by kilacoda** [comment]
I think it might be due to the change to metaclasses in #1572

**Comment by jsonvillanueva** [comment]
> I think it might be due to the change to metaclasses in #1572

#1572 was already included in 0.7.0; the issue is due to my PR that removed the Container ABC #1652. What my PR does is disallow propagating kwargs unnecessarily up to Scene/Mobject. All that needs to be changed is to add the buff parameter to `BraceLabel` and use it by passing the value to `Brace`.

**Comment by Kiran-Raj-Dev** [comment]
The buff keyword argument has been added to `BraceLable` via #1943. Closing.
